oasislmf/__init__.py

```diff
@@ -58,10 +58,11 @@ def find_spec(self, fullname, path=None, target=None):
             import_path = import_path[1]
             for deprecated in self.depricated_modules:
                 if deprecated == import_path or import_path.startswith(deprecated+'.'):
-                    warnings.simplefilter("always")
-                    warnings.warn(
-                        f"imports from 'oasislmf.{deprecated}' are deprecated. Import by using 'oasislmf.{self.depricated_modules[deprecated]}' instead."
-                    )
+                    with warnings.catch_warnings():
```

Comment:
I'm not entirely sure this is necessary and haven't taken the time to confirm that it matters but if I've understood the Python docs this should prevent all warnings being reactivated across the entire application on the next line.